packages/cw-croncat-core/Cargo.toml

```
@@ -32,4 +32,5 @@ serde-json-wasm = { version = "0.4.1", git = "https://github.com/CyberHoward/ser
 [dev-dependencies]
 cosmwasm-schema = { version = "1.0.0" }
 cw-multi-test = { version = "0.16.0", features = ["staking"] }
-cw20-base = { version = "0.13.4", features = ["library"] }
\ No newline at end of file
+cw20-base = { version = "0.13.4", features = ["library"] }
+rusty-hook = "0.11.2"
```

Let's have new lines at the end of the files)
```suggestion
rusty-hook = "0.11.2"

```